vitest.config.ts

```diff
@@ -1,9 +1,13 @@
-import { defineConfig } from 'vitest/config';
+import { defineConfig, configDefaults } from 'vitest/config';
```

Comment:
Not sure what this is doing in this PR, @dpiatek? The submodule is introduced in #218, which is not merged yet, and that PR includes the ignoring that you've added in this commit.

Reply:
ah that's a mistake, didn't delete the submodules when moving between branches so was getting this error locally - will remove!